lib/uptime/check.ex

```diff
@@ -4,9 +4,11 @@ defmodule Uptime.Check do
 
   import Ecto.Changeset
 
+  @type status :: :pending | :success | :failure | :timeout
```

Comment:
me and Tayls had found a way to make this use the Ecto.Enum definition, there is an example somewhere in the auth or portal code base I think